src/nnbench/runner.py

```diff
@@ -187,7 +187,7 @@ def run(
         path_or_module: str | os.PathLike[str],
         params: dict[str, Any] | Parameters | None = None,
         tags: tuple[str, ...] = (),
-        context: Sequence[ContextProvider] = (),
+        context: Sequence[ContextProvider | Context] = (),
```

Comment:
No, I meant `Context | Sequence[ContextProvider]`.

If it's the former, you say `ctx = context` (in L239), if it's the latter, you call `ctx.add(provider)` in the loop over the providers.

Reply:
In this implementation (using `add` we then do not throw any error on multiple values for context key since the `add` (and `update`) methods use `dict.update`, which overwrites keys, under the hood.

Are we fine with that?